x-pack/solutions/security/plugins/security_solution/public/management/links.ts

```diff
@@ -317,6 +323,10 @@ export const getManagementFilteredLinks = async (
     linksToExclude.push(SecurityPageName.cloudDefendPolicies);
   }
 
+  if (isServerless) {
+    linksToExclude.push(SecurityPageName.cloudDefendPolicies);
+  }
```

Comment:
can you write a unit test in the x-pack/solutions/security/plugins/security_solution/public/management/links.test.ts page ?